test-cases/tests/upgrade/n02b-upgrade-rhoam.md

```diff
@@ -48,9 +50,39 @@ Note: If [N09 test case](https://github.com/integr8ly/integreatly-operator/blob/
 
    > Note: do not re-deploy if the workload-web-app is already present in the cluster - check if `workload-web-app` namespace exists in the cluster or not.
 
-   There should be no errors in the command output and product (3scale, SSO) URLS should not be blank. Alternatively, you can check the `Environment` tab in workload-webapp namespace in OpenShift console. See step 7 and 8, you might want to do these pre-upgrade as well.
+   There should be no errors in the command output and product (3scale, SSO) URLS should not be blank. Alternatively, you can check the `Environment` tab in workload-webapp namespace in OpenShift console. See step 8 and 9, you might want to do these pre-upgrade as well.
 
-3. Use the command below to check whether the installPlan exists and is approved.
+3. Prepare corrupted users on the cluster (existence of such users must not break the upgrade)
```

Comment:
Would you want to say something about how these users would need a 3scale account setup also.

Reply:
Hey @Boomatang thanks, I forgot about that piece. In each case the user needs to log into 3scale. And it has to be done before user (scenario 1) or identity (scenario 2) is removed. Correct?

Reply:
Yap I that was what I did locally. The one thing I am not sure if for scenario 1 they need to login. The operator should remove the removed user account from 3scale. But for the easy of creating set up steps it would be best get all users to create 3scale accounts.

Reply:
@Boomatang I added a commit with note on how to log into 3scale. Please take a look when you get a chance. Thanks!